libstuff/libstuff.cpp

```diff
@@ -2615,6 +2615,10 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     if ((int64_t)elapsed > warnThreshold) {
         // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
```

Comment:
Can you remove this todo and leave the part about this removing auth tokens? The linked issue is closed